spring-integration-zip/src/test/java/org/springframework/integration/zip/transformer/UnZipTransformerTests.java

```diff
@@ -16,15 +16,6 @@
 
 package org.springframework.integration.zip.transformer;
 
-import static org.hamcrest.Matchers.containsString;
```

Comment:
Please, don't do such an imports reformat. We really support and require and existing order.